### DISCUSSION TITLE
Will there possibly be a Soyosource2MQTT software?

### DISCUSSION THREAD

**Discussion by thitcher** [softwarecrash/EPEver2MQTT#30] [General]
The EPEver2MQTT works really good, I like it.
Is there a chance to get a Software for Soyosource inverters, too?
That would be really helpful for photovoltaiks in HA.

**Comment by softwarecrash** [comment]
we working on a version for soyosource ;)

**Reply by thitcher** [reply]
Wow! Great!
Will it be possible to control more than one inverter with just one ESP?
I think the inverters don't have an ID that can be changed but you could control them in parallel and then adapt the power to the number of devices using a divider.
That would be absolutely great

**Reply by softwarecrash** [reply]
whats possible or not we dont know, we are working on the core of it at the moment.

**Reply by thitcher** [reply]
This is definitely very good news. Please excuse me, I didn't mean to be too hasty.
The calculations can also be carried out in HA, so that wouldn't be a problem.
I'm definitely very excited about the first release

**Comment by all-solutions** [comment]
> Wow! Great! Will it be possible to control more than one inverter with just one ESP? I think the inverters don't have an ID that can be changed but you could control them in parallel and then adapt the power to the number of devices using a divider. That would be absolutely great

This is exactly how the Soyo works. They are simply "dummies". No possibility of assigning an ID. They only react to the transferred values. With several Soyo, this is then simply divided by the number of devices.
Example: You have a consumption of 600W. If you have one Soyo connected to the ESP, you transmit the value "600", with two Soyo the value "300", with three "200", etc.
The number of connected Soyo is then set via the WebUI, the rest is done by the ESP.

**Reply by Kloana001** [reply]
...hier auch großes Interesse daran...
..sowas steht ganz weit oben auf der Weihnachtswunschliste....
